context/app/static/js/components/detailPage/entityHeader/EntityHeader/style.ts

```diff
@@ -5,7 +5,7 @@ import { headerHeight } from 'js/components/Header/HeaderAppBar/style';
 import { iconButtonHeight } from 'js/shared-styles/buttons';
 
 const StyledPaper = styled(Paper)(({ theme }) => ({
-  position: 'fixed',
+  position: 'sticky',
```

Comment:
This change fixes a strange styling bug that was causing the entity header bar to shift whenever a menu was opened.

Reply:
Nice! I have this in my active branch as well.